core/client/fs/src/test/java/alluxio/client/file/DelegatingFileSystemTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file;
+
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.mock;
+
+import alluxio.AlluxioURI;
+import alluxio.grpc.FileSystemMasterCommonPOptions;
+import alluxio.grpc.MountPOptions;
+import alluxio.grpc.SetAclAction;
+import alluxio.grpc.SetAclPOptions;
+import alluxio.security.authorization.AclEntry;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Tests {@link DelegatingFileSystem}.
+ */
+public class DelegatingFileSystemTest {
+    private FileSystem mMockFileSystem;
```

Comment:
This file should be updated with the correct indenting. (2 spaces instead of 4 spaces)